README.md

```diff
@@ -47,7 +47,7 @@ You can access these pages on your computer using one of the following clients:
   - [tldroid](https://github.com/hidroh/tldroid), available on
     [Google Play](https://play.google.com/store/apps/details?id=io.github.hidroh.tldroid)
 - [Bash client](https://github.com/raylee/tldr)
-- [C++ client](https://github.com/tldr-pages/tldr-cpp-client): `brew install tldr-pages/tldr/tldr`
+- [C++ client](https://github.com/tldr-pages/tldr-cpp-client): `brew install tldr`
```

Comment:
@ericbn I've addressed this in e20cde3. Once this branch is rebased, we can focus on the change in the other discussion thread.